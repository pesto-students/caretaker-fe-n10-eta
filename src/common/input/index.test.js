import React from "react";
import renderer from "react-test-renderer";
import TextInput from "./index";

it("ActiveMark renders correctly", () => {
  const tree = renderer.create(<TextInput />).toJSON();
  expect(tree).toMatchSnapshot();
});
