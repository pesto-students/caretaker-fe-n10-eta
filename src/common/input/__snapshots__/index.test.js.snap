// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveMark renders correctly 1`] = `
<input
  className="ant-input"
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  type="text"
  value=""
/>
`;
